packages/shared/src/strings.ts

```diff
@@ -5,6 +5,10 @@ export type CamelCaseToSnakeCase<S extends string> =
         : ''}${Lowercase<T>}${CamelCaseToSnakeCase<U>}`
     : S;
 
+type CamelCase<T extends string> = T extends `${infer F}${infer R}`
```

Comment:
isnt this PascalCase?

Reply:
i have no idea lol, copied the type from another repo. will change